docker/Dockerfile

```diff
@@ -0,0 +1,128 @@
+# Build from https://github.com/osrf/docker_images/tree/master/ros/indigo/indigo-ros-base
+FROM ros:indigo
+MAINTAINER Dmitry Kiselev <dmitry@endpoint.com>
+
+# nvidia-docker hooks
+LABEL com.nvidia.volumes.needed="nvidia_driver"
+ENV PATH /usr/local/nvidia/bin:${PATH}
+ENV LD_LIBRARY_PATH /usr/local/nvidia/lib:/usr/local/nvidia/lib64:${LD_LIBRARY_PATH}
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install misc system dependencies.
+RUN \
+  apt-get update && \
+  apt-get -y install --no-install-recommends \
+    bc \
+    lsb-core \
+    x-window-system \
+    dbus-x11 \
+    dconf-cli \
+    wget \
+    curl \
+    tmux \
+    apt-transport-https \
+    libglu1-mesa \
+    socat \
+    libnss3-tools \
+    gir1.2-atspi-2.0 \
+    binutils \
+    mesa-utils \
+    mesa-utils-extra \
+    module-init-tools \
+    gdebi-core \
+    tar \
+    libfreeimage3 \
+    xdg-utils \
+    onboard \
+    git \
+    libexif12 \
+    libexif-dev \
+    vim \
+    nano \
+    mc
```

Comment:
you probably want a `&& rm -rf /var/lib/apt/lists/*` here, as each `RUN` command marks a commit when building a docker image.